src/protocols/secure_channel/CASESession.h

```diff
@@ -58,19 +58,21 @@ constexpr size_t kCASEResumptionIDSize = 16;
 #define CASE_EPHEMERAL_KEY 0xCA5EECD0
 #endif
 
-struct CASESessionSerialized;
+struct CacheableCASEParameters;
```

Comment:
This structure is not needed anymore. We are able store the binary data in the persistent storage. So no need to base64 encode the structure. We can also remove the Serialize/Deserialize functions from this class.

Reply:
Done.